Lib/unittest/mock.py

```diff
@@ -1633,11 +1640,12 @@ def __exit__(self, *exc_info):
                 # needed for proxy objects like django settings
                 setattr(self.target, self.attribute, self.temp_original)
 
-        del self.temp_original
-        del self.is_local
-        del self.target
+        self.is_started = False
         exit_stack = self._exit_stack
         del self._exit_stack
+        del self.is_local
+        del self.temp_original
+        del self.target
```

Comment:
Please undo this change: `self.is_started = False` should be just before the return, and that feels like it should be the only change here.

Reply:
Undone all 3